Homebrew/Homebrew.munki.recipe.yaml

```diff
@@ -28,28 +28,40 @@ Process:
         installcheck_script: |
           #!/bin/zsh
 
+          autoload is-at-least
+
           installed=1
           not_installed=0
           target_version="%version%"
 
+          # Get the current user
+          current_user="$(/usr/bin/stat -f%Su /dev/console)"
+
+          # If we're at the login window, Homebrew's .pkg will fail to install
+          if [[ $current_user == "root" || $current_user == "_mbsetupuser" || $current_user == "loginwindow" ]]; then
+            /bin/echo "No user account is logged in, so considering \"installed\" for now."
+            exit "$installed"
+          fi
+
```

Comment:
Let's borrow this logic instead. Assumes the admin correctly set the plist value with an existing user. Acknowledges login window installs should work and still ends up running `brew --version` as the user, not root. 

https://github.com/Homebrew/brew/blob/4ee6e96bdf857b3c50290bf032a9645b3bf293f1/package/scripts/postinstall#L66-L74

Reply:
Done! https://github.com/autopkg/nstrauss-recipes/pull/100/commits/d11ef65c36cec8cc1ec421ad0ddff2fe05d22d9d